build.assets/Dockerfile

```diff
@@ -44,9 +44,9 @@ RUN git clone --depth=1 https://github.com/PJK/libcbor.git -b v0.10.2 && \
 
 # Install openssl.
 # install_sw install only binaries, skips docs.
-RUN git clone --depth=1 https://github.com/openssl/openssl.git -b openssl-3.0.10 && \
+RUN git clone --depth=1 https://github.com/openssl/openssl.git -b openssl-3.0.11 && \
     cd openssl && \
-    [ "$(git rev-parse HEAD)" = '245cb0291e0db99d9ccf3692fa76f440b2b054c2' ] && \
+    [ "$(git rev-parse HEAD)" = '6ba3884c3235e1bb474b379026087f8216afacf4' ] && \
```

Comment:
Why do we check the output of `git rev-parse HEAD` instead of just doing an additional (noop, ideally) `git checkout <commit hash>`?

Reply:
I don't think there's a particular reason to choose `git rev-parse` vs `git checkout`, either would do. `git clone -b <hash>` would probably do the trick too, although being less readable.

Reply:
I don't believe you can `git clone` a specific commit, unfortunately 😞

Reply:
You can get a tarball for a specific commit from github tho 👀